src/content/energy-consumption/index.md

```diff
@@ -7,61 +7,44 @@ sidebar: true
+PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in more powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).
 
-- 25,000 transactions per second.
-- `100,000 / 25,000 = 4` seconds to process 100,000 transactions.
+Read more detailed information about [how PoS is implemented in Ethereum](/developers/docs/consensus-mechanisms/pos) and how it compares to PoW.
 
-We can also estimate Ethereum's energy expenditure per second, making a conservative estimate that 10,000 active validators are securing the network (there are over [250,000 validators on the Beacon Chain](https://beaconscan.com/) at the moment, but many validators can operate on a single node. Currently, there are estimated to be 3,000-4,000 individual nodes, so 10,000 is a conservative estimate for post-merge):
+## Proof-of-stake energy expenditure {#proof-of-stake-energy}
 
-`1.44kWh daily usage * 10,000 network nodes = 14,400kWh` per day.
-There are 86,400 seconds in a day, so `14,400 / 86,400 = 0.1667 kWh` per second.
+Estimates based on the current Beacon Chain suggest that the merge to PoS could result in a 99.95% reduction in total energy use, with PoS being ~2000x more energy-efficient than PoW. The energy expenditure of Ethereum will be roughly equal to the cost of running a modest laptop for each node on the network.
 
-If we multiply that by the amount of time it takes to process 100,000 transaction: `0.1667 * 4 = 0.667 kWh`.
+Many articles estimate "per-transaction" energy expenditure to compare blockchains to other industries. The benefit of this is that it is easy to understand, but the energy required to mine a block is independent of the number of transactions within it. A per transaction unit of energy expenditure implies that fewer transactions would lead to smaller energy expenditure, and vice-versa, which is not the case. A per-transaction estimate is highly dependent upon how a blockchain's transaction throughput is defined, and tweaking this definition can be gamed to make the value seem larger or smaller.
```

Comment:
```suggestion
Many articles estimate "per-transaction" energy expenditure to compare blockchains to other industries. The benefit of this is that it is easy to understand, but the energy required to mine a block is independent of the number of transactions within it. A per transaction unit of energy expenditure implies that fewer transactions would lead to smaller energy expenditure and vice-versa, which is not the case. A per-transaction estimate is highly dependent upon how a blockchain's transaction throughput is defined, and tweaking this definition can be gamed to make the value seem larger or smaller.
```